docs/snap-clone.md

```diff
@@ -116,3 +120,100 @@ csi-cephfs-pvc       Bound    pvc-1ea51547-a88b-4ab0-8b4a-812caeaf025d   1Gi
 cephfs-pvc-clone     Bound    pvc-b575bc35-d521-4c41-b4f9-1d733cd28fdf   1Gi        RWX            csi-cephfs-sc  39s
 cephfs-pvc-restore   Bound    pvc-95308c75-6c93-4928-a551-6b5137192209   1Gi        RWX            csi-cephfs-sc  55m
 ```
+
+## Create RBD snapshot and Clone Volume
+
+In the `examples/rbd` directory you will find two files related to snapshots:
+[snapshotclass.yaml](../examples/rbd/snapshotclass.yaml)
+and [snapshot.yaml](../examples/rbd/snapshot.yaml)
+
+Once you created RBD PVC, you'll need to customize `snapshotclass.yaml` and
+make sure the `clusterid` parameter matches `clusterid` mentioned in the
+storageclass from which the PVC got created.
+If you followed the documentation to create the rbdplugin, you shouldn't
+have to edit any other file.
+
+After configuring everything you needed, deploy the snapshotclass:
+
+### Create RBD-snapshotclass
+
+```bash
+kubectl create -f snapshotclass.yaml
+```
+
+### Verify that the snapshot class was created
+
+```console
+$ kubectl get volumesnapshotclass
+NAME                      AGE
+csi-rbdplugin-snapclass   4s
+```
+
+### Create rbd-snapshot
+
+```bash
+kubectl create -f snapshot.yaml
+```
+
+### Verify if your volume snapshot has successfully been created
+
+```console
+$ kubectl get volumesnapshot
+NAME               AGE
+rbd-pvc-snapshot   6s
+```
+
+### Check the status of the snapshot
+
+```bash
```

Comment:
this contains output, not only commands, so it should be marked `console`

Reply:
done.